easybuild/easyconfigs/c/COMSOL/COMSOL-6.0.0.318.eb

```diff
@@ -10,9 +10,10 @@ problems.
 
 toolchain = SYSTEM
 
-sources = ['COMSOL%(version_major)s%(version_minor)s_dvd.iso']
+# Note: sources from COMSOL are often named poorly. You can view the exact version after installing.
```

Comment:
Perhaps change this to "view the exact version by looking in the xxx file in the iso"